playbooks/provision_openstack_instance.yml

```diff
@@ -0,0 +1,34 @@
+---
+- name: CVP 1164
+  hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("yes") }}'
+  gather_facts: '{{ gather_facts | default("no") }}'
+
+  tasks:
+
+    - name: provision a key pair
+      os_keypair:
+        state: present
+        name: ansible_key
+      register: keypair_output
+
+    - debug:
```

Comment:
@LikhithaEda You can remove this task. Debugs are not needed

Reply:
The playbook does not run if I remove the debug.